controllers/cruisecontroltask_types.go

```diff
@@ -15,44 +15,34 @@
 package controllers
 
 import (
-	kafkav1beta1 "github.com/banzaicloud/koperator/api/v1beta1"
-	"github.com/banzaicloud/koperator/pkg/scale"
-)
+	corev1 "k8s.io/api/core/v1"
 
-type CruiseControlOperation int8
-
-const (
-	OperationAddBroker CruiseControlOperation = iota
-	OperationRemoveBroker
-	OperationRebalanceDisks
+	kafkav1alpha1 "github.com/banzaicloud/koperator/api/v1alpha1"
```

Comment:
**Recommended**: This **SHOULD** be aliased as `koperatorv1alpha1` IMO to distinguish from upstream Kafka types.

```suggestion
	koperatorv1alpha1 "github.com/banzaicloud/koperator/api/v1alpha1"
```

This requires updates throughout the file.